cmake/chimeraFunctions.cmake

```diff
@@ -24,7 +25,7 @@ function(add_chimera_binding)
     # Unparsed arguments can be found in variable ARG_UNPARSED_ARGUMENTS.
     set(prefix binding)
     set(options DEBUG EXCLUDE_FROM_ALL)
-    set(oneValueArgs TARGET MODULE CONFIGURATION DESTINATION BINDING)
+    set(oneValueArgs TARGET MODULE CONFIGURATION DESTINATION BINDING GENERATED_BINDING_SOURCES)
```

Comment:
Please rename this to `GENERATED_SOURCES_VAR` to help distinguish it as a string variable name rather than a value.

Also, does this differ in a meaningful way from the result of  ```get_target_property(GENERATED_SOURCES my_chimera_target SOURCES)```

Reply:
> Please rename this to `GENERATED_SOURCES_VAR` to help distinguish it as a string variable name rather than a value.

Sounds good to me



> Also, does this differ in a meaningful way from the result of `get_target_property(GENERATED_SOURCES my_chimera_target SOURCES)`

I believe it would return all the sources including the sources specified as `EXTRA_SOURCES` for this function. This variable is to replace [this hassle](https://github.com/personalrobotics/dartpy/blob/3535357b354b48919d9912c24aff37eff8d6e711/CMakeLists.txt#L144-L147) in the user side.

Reply:
But there, we would be able to use the `EXTRA_SOURCES`, since we need them anyway to compile, right?

Reply:
That's correct.

I think this variable still useful as it's an easier way than using the extra function. I'm okay to remove this variable and document about how to extract the source files.

The bigger motivation behind this variable is to simplify the chimera setting on the user side (#317). If we extend `add_chimera_binding()` to create the custom target just for generating bindings, then this variable wouldn't be much useful.

Reply:
> The bigger motivation behind this variable is to simplify the chimera setting on the user side (#317). If we extend `add_chimera_binding()` to create the custom target just for generating bindings, then this variable wouldn't be much useful.

If this sounds good to you, then I think we can revert this PR (or when #317 is resolved).